lib/model/query/sessions.js

```diff
@@ -14,19 +14,31 @@ const { generateToken, isValidToken } = require('../../util/crypto');
 const { unjoiner } = require('../../util/db');
 const { construct } = require('../../util/util');
 const Option = require('../../util/option');
+const DEFAULT_SESSION_LIFETIME = 24 * 60 * 60;
 
-const aDayFromNow = () => {
-  const date = new Date();
-  date.setDate(date.getDate() + 1);
-  return date;
+const create = (actor, expiresAt) => ({ one }) => {
+  const SESSION_LIFETIME = parseInt(process.env.ODKCENTRAL_SESSION_LIFETIME, 10) || DEFAULT_SESSION_LIFETIME;
+  const expirySqlFragment = (expiresAt instanceof Date ? sql`${expiresAt.toISOString()}` : sql`statement_timestamp() + ${SESSION_LIFETIME + ' s'}::interval`);
```

Comment:
ah another way to get timestamp in postgresql 😭

In the codebase, we already have so many ways to get timestamp see https://github.com/getodk/central-backend/issues/1439

I think `now()` or `current_timestamp()` would be better here